### ISSUE TITLE
Suggestion: Client Side Spectator Mode

### ISSUE THREAD

**Issue by dk88711** [Earthcomputer/clientcommands#82]
To able to look around, noclip, just like vanilla spectator mode. So that I can find air gaps to fill, dark caves to light, etc. for making a mob farm. Is it possible with fabric API?

**Comment by Earthcomputer** [comment]
Good idea! I'm pretty sure I've implemented this before actually, I don't know where that code went.

**Comment by Mhowser** [comment]
It would great if we could use F3 + N to quickly switch just like vanilla.

**Comment by Anunayj** [comment]
Can always use tweakroo Freecam. Don't feel like there is any point in having 10 different mods for same thing. Though can be useful for some cases I guess.

**Comment by Earthcomputer** [comment]
Huh. I was not aware that Tweakeroo had this feature.

**Comment by leumasme** [comment]
Couldn't find the feature on the CurseForge Page.

**Comment by hube12** [comment]
it has been available for a while now, please dont hesitate to download from masa site directly to have the latest

**Comment by dk88711** [comment]
Seems Tweakeroo Freecam is not as powerful as the real spectator mode, caves don't render if there are opaque blocks in between. It's development version though.

**Comment by James103** [comment]
I believe this issue can be closed now since Tweakeroo's freecam mode is now a fully-featured free-cam mode, making freecam mode implementations in Client Commands redundant.

**Comment by Earthcomputer** [comment]
Sorry but this issue tracks the suggestion for *clientcommands*, and not affected by whether it's implemented on other mods.
